src/Ryujinx.Ui.Common/DiscordIntegrationModule.cs

```diff
@@ -66,17 +66,17 @@ public static void SwitchToPlayingState(string titleId, string titleName)
             {
                 Assets = new Assets
                 {
-                    LargeImageKey  = "game",
+                    LargeImageKey = "game",
                     LargeImageText = titleName,
-                    SmallImageKey  = "ryujinx",
+                    SmallImageKey = "ryujinx",
                     SmallImageText = Description,
                 },
-                Details    = $"Playing {titleName}",
-                State      = (titleId == "0000000000000000") ? "Homebrew" : titleId.ToUpper(),
+                Details = $"Playing {titleName}",
+                State = (titleId == "0000000000000000") ? "Homebrew" : titleId.ToUpper(),
                 Timestamps = Timestamps.Now,
-                Buttons    = new Button[]
+                Buttons = new[]
                 {
-                    new Button()
+                    new Button
                     {
                         Label = "Website",
                         Url   = "https://ryujinx.org/"
```

Comment:
```suggestion
                        Url = "https://ryujinx.org/",
```

tailing `,` is missing after next `}`